README.md

```diff
@@ -1,114 +1,80 @@
 [![Open in GitHub Codespaces](
   https://img.shields.io/badge/Open%20in%20GitHub%20Codespaces-333?logo=github)](
   https://codespaces.new/dwave-examples/employee-scheduling?quickstart=1)
-[![Linux/Mac/Windows build status](
+<!-- [![Linux/Mac/Windows build status](
   https://circleci.com/gh/dwave-examples/employee-scheduling.svg?style=shield)](
-  https://circleci.com/gh/dwave-examples/employee-scheduling)
+  https://circleci.com/gh/dwave-examples/employee-scheduling) -->
 
-# Building an Employee Schedule
+# Employee Scheduling Demo
 
-Building a schedule for employees can be an extremely complex optimization
-problem in which managers must balance employee preferences against schedule
-requirements. In this example, we show how a discrete quadratic model (DQM) 
-and a constrained quadratic model (CQM) can be used to model this problem and
-how the hybrid solvers available in Leap can optimize over these competing 
-scheduling and preference needs.
+Employee scheduling is a common industry problem that can often become complex
+due to real-world constraints and requirements. In this example, we demonstrate
+a scheduling scenario with a variety of employees and rules.
 
-## Usage
+## Running the Demo
 
-To run the CQM demo, type the command:
+To run the demo, open the terminal and run `python app.py`. Open a web browser
+and copy the URL from the terminal to load the demo in the browser.
 
-```python demo.py```
+Once we have set all of the options to our satisfaction, click the "Solve CQM"
+button on the left. In the terminal we see status updates for "Building CQM..."
+and "Submitting CQM...".
 
-A prompt will appear asking for the number of employees:
+Once the problem has completed, the best solution returned will be displayed in
+place of "Employee Availability". Click back to the original availability using
+the tabs at the top of the schedule card.
 
-```Enter number of employees:```
+The solution returned will be either the best feasible solution (if a feasible
+solution is found) or the best infeasible solution (if no feasible solution is
+found). If an infeasible solution is found, scrolling down will show the list of
+constraints that were violated.
 
-Type the number of employees to be considered and hit `Enter`. Note that the
-CQM solver can take up to 5000 variables, or employees for this problem.
+## Introducing the Demo
 
-A second prompt for the number of shifts will appear:
+When the demo opens in the web browser, there are a few options on the left and
+a chart of employee availability in the center.
 
-```Enter number of shifts:```
+![Screen Image](assets/demo_image.png)
 
-Type the number of shifts and hit `Enter`. Note that we should have at least as
-many employees as there are shifts.
+### Employee Availability
```

Comment:
```suggestion
```

The following text will be under the template's ``Inputs`` header three (if you go with the previous suggested changes)